server/storage.py

```diff
@@ -92,6 +92,11 @@ def retrieve_team_uuid(program, team_number):
                 'last_time': datetime.now(timezone.utc),
                 'preferences': {},
                 'last_video_uuid': '',
+                'videos_uploaded_today': 0,
+                'datasets_downloaded_today': 0,
+                'datasets_created_today': 0,
+                'models_downloaded_today': 0,
```

Comment:
I'm not sure we need to limit models downloaded. It doesn't need to zip it up like it does for a dataset.